### ISSUE TITLE
When editing a component in the designer, the grey modal background does not go away on submit

### ISSUE THREAD

**Issue by mozfet** [mozfet/ExpertBox#7]
The modal goes away, leaving the grey semi transparent background. The background goes away on the first click.

**Comment by mozfet** [comment]
when editing component in designer, the grey background only stays on submit, not on cancel. It thus works correctly for cancel, what is different between submit and cancel?

**Comment by mozfet** [comment]
To keep the UI simple we have a rule that only one modal can be open at a time. This could thus be fixed this by removing all modal-overlay instances every time after a modal is closed.
```
//close the modal
$('#'+instance.data.id).modal('close');
$('.modal-overlay').remove();
```

fixed in v1.1.3
